services/ui-src/src/components/export/ExportedReportFieldTable.tsx

```diff
@@ -26,8 +26,26 @@ export const ExportedReportFieldTable = ({ section }: Props) => {
   const { tableHeaders } = verbiage;
 
   const pageType = section.pageType;
-  const formFields =
+  let formFields =
     pageType === "drawer" ? section.drawerForm?.fields : section.form?.fields;
+
+  const renderNotReportingFields = (field: FormField | FormLayoutElement) => {
+    report?.fieldData[field.id]?.value === "Yes"
```

Comment:
Would like to avoid using `!`. Suggested refactor:
```
formFields = report?.fieldData[field.id]?.value === "Yes"
  ? [field, ...(formFields || [])]
  : section.form?.fields;
```

Reply:
thank you! just committed that suggestion 💯